### DISCUSSION TITLE
一直显示登录节点失败

### DISCUSSION THREAD

**Discussion by lotussssss111** [natfrp/wiki#460] [使用问题] [answered]
登录节点失败: read tcp 172.20.13.53:53487->42.202.37.75:7000: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond., 请检查网络连接
是因为校园网吗？还是别的原因

**Comment by fakeboboliu** [comment, marked as answer]
我们已经为此问题更新了文档内容，您可以看：https://doc.natfrp.com/faq/network.html#zoned-network-restriction

**Reply by lotussssss111** [reply]
感谢，之前怎么都不行，今天换成流量就可以了
